mizar/obj/droplet.py

```diff
@@ -37,7 +39,7 @@ def __init__(self, name, obj_api, opr_store, spec=None):
         self.store = opr_store
         self.ip = ""
         self.mac = ""
-        self.phy_itf = 'eth0'
+        self.phy_itf = os.getenv("INET")
```

Comment:
Let's set the default value as "eth0" here:
self.phy_itf=os.getenv("INET") if os.getenv("INET") is not None else "eth0"